CONTRIBUTOR.md

```diff
@@ -2,52 +2,44 @@ Faire/mjml-react is one of [Faire](https://www.faire.com)'s first open source pr
 - publishing the next version package to [NPM](https://www.npmjs.com/package/@faire/mjml-react)
 
-This process is triggered via Github actions by pushing to a release branch.
+This process is triggered via Github actions by pushing to a release branch. The next version is determined from the most recent commit with a version tag that corresponds to the branch (i.e. `v3.0.1.` vs. `v.3.0.1-alpha.1`, see more on prereleases in Branch Organization).
 
-When pull requests are merged, the changes are squashed into one commit which uses the pull request title as the message. Thus, in order for the merged code to trigger a version bump and release, <b><u>your pull request title must be formatted in one of the following ways</u></b>:
+When pull requests are merged, the changes are squashed into one commit which uses the pull request title as the default message. In order for the merged code to trigger a version bump and release, <b><u>your pull request title should be formatted in one of the following ways</u></b>:
 
 | Commit message                                                                                                                                                                                   | Release type                                                                                                               |
 | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------- |
 | `fix(pencil): stop graphite breaking when too much pressure applied`                                                                                                                             | ~~Patch~~ Fix Release                                                                                                      |
 | `feat(pencil): add 'graphiteWidth' option`                                                                                                                                                       | ~~Minor~~ Feature Release                                                                                                  |
 | `perf(pencil): remove graphiteWidth option`<br><br>`BREAKING CHANGE: The graphiteWidth option has been removed.`<br>`The default graphite width of 10mm is always used for performance reasons.` | ~~Major~~ Breaking Release <br /> (Note that the `BREAKING CHANGE: ` token must be in the footer/body of the pull request) |
+| `chore(pencil): remove graphiteWidth option`                                                                                                                                                     | No Release (no version update)                                                                                             |
```

Comment:
```suggestion
| `chore(pencil): document graphiteWidth option`                                                                                                                                                     | No Release (no version update)                                                                                             |
```
? a removal should probably at least be a patch

Reply:
oh whoops didn't save in my editor, I have `chore(docs): update README.MD`